src/components/UploaderToolDialog/PackageTable.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  IconButton,
+  Stack,
+  styled,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@mui/material";
+import { Link } from "react-router-dom";
+import { ReactComponent as DownloadIcon } from "../../assets/icons/download_icon.svg";
+import packageConfig from "../../config/PackageTableConfig";
+
+const StyledTableContainer = styled(TableContainer)({
+  borderRadius: "8px",
+  border: "1px solid #6B7294",
+  position: "relative",
+  overflow: "hidden",
+  background: "#FFF",
+  "& .MuiTableHead-root .MuiTableCell-root, .MuiTableBody-root .MuiTableRow-root:not(:last-of-type) .MuiTableCell-root":
+    {
+      borderBottom: "1px solid #6B7294",
+    },
+  "& .MuiTableCell-root:not(:last-of-type)": {
+    borderRight: "1px solid #6B7294",
+  },
+});
+
+const StyledTableHead = styled(TableHead)({
+  "& .MuiTableCell-head": {
+    fontFamily: "'Nunito', 'Rubik', sans-serif",
+    fontWeight: 700,
+    fontSize: "16px",
+    lineHeight: "19.6px",
+    color: "#083A50",
+    padding: "11px 15px",
+  },
+});
+
+const StyledTableBody = styled(TableBody)({
+  "& .MuiTableCell-body": {
+    fontFamily: "'Nunito', 'Rubik', sans-serif",
+    fontWeight: 400,
+    fontSize: "16px",
+    lineHeight: "24px",
+    color: "#083A50",
+    padding: "8px 16px",
+  },
+});
+
+const StyledDownload = styled(Link)({
+  fontFamily: "'Nunito', 'Rubik', sans-serif",
+  fontWeight: 700,
+  fontSize: "16px",
+  lineHeight: "19.6px",
+  color: "#365F71",
+  textDecoration: "underline",
+  marginRight: "8px",
+});
+
+const PackageTable = () => (
```

Comment:
When opening the dialog on the Model Navigator page, the spacing within the table gets messed up. Can you take a look at it? 

<details><summary>Normal vs. Model Navigator</summary>
<p>
<img width="747" alt="Screenshot 2024-08-13 at 10 02 40 AM" src="https://github.com/user-attachments/assets/d61e6608-21cf-4017-b96a-b6e5920e280c">
</p>
<p>
<img width="747" alt="Screenshot 2024-08-13 at 10 02 49 AM" src="https://github.com/user-attachments/assets/503413f4-f8b5-4fed-a1ff-496428bb6562">
</p>
</details> 

P.S. if model navigator won't load the model, switch `REACT_APP_DEV_TIER` to `DEV`

Reply:
Ah forgot to check this. Thanks for catching this!